### PR TITLE
feat (User Management): add query params search and include_settings to list_users operation

### PR DESCRIPTION
**Pull request by PureMathematics**
## PR summary

Update Include_settings and substring search functionality (User Management) 

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information
the current issue for updating SDK https://github.ibm.com/IAM/uum/issues/5882 
the previous issue for the user management code change https://github.ibm.com/IAM/uum/issues/4948

### CODE REVIEW

File: examples/user-management.v1.test.js [hunk 1]
```diff
@@ -177,9 +177,74 @@ describe('UserManagementV1', () => {
     } catch (err) {
       console.warn(err);
     }
-
     // end-list_users
   });
+  test('listUsers request include_settings example', async () => {
```
Comment (padamstx) [inline examples/user-management.v1.test.js:182]: same comment as in Go SDK, one code fragment per operation :)

**Review by padamstx — CHANGES_REQUESTED** [review]
Change look good, but same comments as the other PRs:
1. please use the git commit "-s" option.
2. one code fragment per operation within the examples code

File: examples/user-management.v1.test.js [hunk 2]
```diff
@@ -163,6 +163,8 @@ describe('UserManagementV1', () => {
 
     const params = {
       accountId: accountId,
+      includeSettings: true,
+      search: 'state%3AACTIVE',
```
Comment (padamstx) [inline examples/user-management.v1.test.js:167]: same comment about URL-encoding

**Review by padamstx — COMMENTED** [review]
LGTM, just have the question about URL-encoding

**Review by padamstx — APPROVED** [review]
LGTM

**Comment by PureMathematics** [comment]
integration test
node-integration

**Comment by PureMathematics** [comment]
examples test
node-example

**Pull request merged**